xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-test/xwiki-platform-mentions-test-docker/src/test/it/org/xwiki/mentions/test/ui/MentionsIT.java

```diff
@@ -141,4 +229,31 @@ private void runAsSuperAdmin(TestUtils setup, RunnableErr actions) throws Except
         setup.loginAsSuperAdmin();
         actions.run();
     }
+
+    private <T> T retry(int times, Callable<T> callable) throws Exception
```

Comment:
That's not good, you shouldn't handle the test retry inside your own test. Test retry should be done at the execution level: we already have what's needed for it, it's just that we don't enable it usually for timeout reason.